src/test/java/com/google/devtools/build/lib/dynamic/DynamicSpawnStrategyTest.java

```diff
@@ -419,7 +419,7 @@ private Spawn newCustomSpawn(String mnemonic, ImmutableMap<String, String> execu
         executionInfo,
         EmptyRunfilesSupplier.INSTANCE,
         action,
-        ResourceSet.create(1, 0, 0));
+        ResourceSet.create(1, 0, null, 0));
```

Comment:
A number of these tests could use the two-parameter overload.

Reply:
Looks like many actually can't, because they also specify a nonzero localTestCount. In fact, looks like this was the only place where I could use `ResourceSet.createWithRamCpu` instead.